Functions/CatalogItems/Get-RsFolderContent.ps1

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Get-RsFolderContent
```

Comment:
I think you need to update FunctionsToExport  in ReportingServicesTools.psd1 and also add the alias in AliasesToExport